### ISSUE TITLE
When I am training, the memory usage is very large. Is this normal？

### ISSUE THREAD

**Issue by HIT16S** [EherSenaw/ViDT_colab#1]
您好，我使用ViDT在coco上做训练时，batch_size=2时，显存占用到了17GB左右，不知道为什么会占用如此大的显存，您碰到过这个问题吗？或者这个问题可能的原因是什么？谢谢~

**Comment by EherSenaw** [comment]
(As I'm not familiar with chinese language, I translated your comment with Google Translate, so if I misunderstood, let me know with English)
I only tested `batch_size=16`, so I would not clearly say that behavior is weird or not.

But in my `Colab Pro` environment (assumed to have 25GB RAM), `batch_size=16` had run without any exception.
For me, the GPU RAM util graph showed almost half of the limit, increasing very slightly per 5000 epochs.

Therefore, I think `batch_size=2` might still work well.

p.s. I ran training with `--backbone_name swin_nano` option, have you checked it?
